CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 * Stringify CanvasGradient, CanvasPattern and ImageData like browsers do. (#1639, #1646)
 * Add missing include for `toupper`.
 * Throw an error instead of crashing the process if `getImageData` or `putImageData` is called on a PDF or SVG canvas (#1853)
+* Bump `simple-get` to 4.0.1 to fix a depentency vulnerability
```

Comment:
```suggestion
* Bump `simple-get` to 4.0.1 to fix a dependency vulnerability
```

Reply:
Yeah, I saw that sometime afterward... Was going to correct it when the decision is made which version to go with if simple-get were to release 3.1.1 or something... Thank you for thoroughness :)